scripts/codegen/CAN/cantx_codegen.py

```diff
@@ -136,7 +136,7 @@ def __init_functions(self, function_prefix):
                         function_prefix, signal.snake_name.upper(), self._sender.capitalize(), signal.type_name),
                         '',
                         '''\
-    if (isnanf(value))
+    if (isnan(value))
```

Comment:
I think we should keep this as isnanf since this code is auto-generated, and compiled for our arm processor